### DISCUSSION TITLE
Forward Sensor not showing IP address or Description after 2.3.120 upgrade

### DISCUSSION THREAD

**Discussion by pimpernoni** [Security-Onion-Solutions/securityonion#7902] [Unsupported Versions]
Curious loss of Forwad Sensor Grid detail on recent upgrade to 2.3.120. Distributed deployment with Manager on 10.0.10.0/24 net and single remote Forward sensor connected to 192.168.1.0/24 via IPSEC with full connectivity working between Master/Sensor before upgrade, with Grid showing Sensor details.

Checked the following: so-status OK on both; certificates accepted by Master for the Sensor; Can ping and SSH to Sensor and Master respectively from each network, and I seem to be getting all the data from the Forward Sensor network into the Master. Tried soup for both again, and no problems reported. 

Can't seem to find where the Grid picks up the detail about the various hosts. Problem seems more cosmetic, but I'd rather not see a "Grid!" alert on the SOC. Any suggestions where I can look?

Thanks for pointing me in the right direction.

**Comment by dougburks** [comment]
Is the forward node able to connect to the manager on all ports listed at https://docs.securityonion.net/en/2.3/firewall.html#node-communication, specifically 443?

**Reply by pimpernoni** [reply]
Doug,
Thanks for the suggestion. I have gone through all the ports and the Forward Sensor can access them, even the 443 port via curl, but the sensor is not allowed because it is not identified as an analyst?

However, all the traffic seems to be captured to the Master (running Forward on a separate remote network using IPSEC). The sensor shows status OK, and time is updated. It is just curious and frustrating that I can't seem to find where the Grid gets the data for the Forward Sensor.

![image](https://user-images.githubusercontent.com/78883311/169167816-a8873ff0-85aa-45c8-8a6b-8e80c6541331.png)

**Reply by dougburks** [reply]
> I have gone through all the ports and the Forward Sensor can access them, even the 443 port via curl, but the sensor is not allowed because it is not identified as an analyst?

I'm not sure I understand this statement. When the sensor joined the manager, the manager should have created a firewall rule to allow the sensor to connect to port 443. Run the following on the manager and see if the correct IP address for the sensor is allowed:
```
sudo iptables -nvL |grep 443
```

**Reply by pimpernoni** [reply]
Doug,
Thanks for the guidance, I was hoping that upgrade to 2.3.130 would resolve this issue, but it still exists where the forward sensor does not show up on the Grid, however, all data seems to be captured and the sensor upgraded and is full status.

Here is the output from the abover recommend iptables query:

![image](https://user-images.githubusercontent.com/78883311/173667911-6514986a-5ba2-4b94-89e9-b197fd9c10b1.png)

**Reply by dougburks** [reply]
Is there any network equipment between the forward node and the manager like a firewall, NAT, or something else that might interfere with the traffic?

**Comment by n0th1nt0lOOz** [comment]
For me, I can't see anything in Grid but still have alert
![image](https://user-images.githubusercontent.com/28757980/167337241-2f04b8b5-3835-4b06-b4fb-df578ebbe52a.png)

**Reply by dougburks** [reply]
@tientmse62290 From https://github.com/Security-Onion-Solutions/securityonion/discussions/1720:
> Start a new discussion instead of replying to somebody else's discussion
Please search to see if you can find similar discussions that may help you. However, in order to avoid confusion, please do NOT reply to somebody else's discussion with your own issue. Instead, please start a new discussion and in that new discussion you can provide a hyperlink to the related discussion.
